src/homework5/runner_and_decorator.py

```diff
@@ -0,0 +1,86 @@
+"""Функция runner.
+
+Вызывает функции, написанные в homework4:
+a. runner() – все фукнции вызываются по очереди.
+b. runner(‘func_name’) – вызывается только функцию func_name.
+c. runner(‘func’, ‘func1’...) - вызывает все переданные функции.
+"""
+
+
+class Runner:
+    """Define class Runner."""
+
+    from mypkg.cities import cities
+    from mypkg.dict_comprehensions import dict_compr
+    from mypkg.euclid_gcd import euclid_gcd
+    from mypkg.languages import lang
+    from mypkg.num_lists_1 import num_lists_1
+    from mypkg.num_lists_2 import num_lists_2
+    from mypkg.words import words
+
+
+LOG_FILE = 'log.txt'
+
+# stores 'function_name':<function_object> from imported modules
+fn_fn = {fn_name: getattr(Runner, fn_name) for fn_name in dir(Runner)
```

Comment:
not holistic solution

Reply:
Have no idea what do you mean... But I tried to rewrite module a bit, so may be solution became holistic.